.github/workflows/e2e.yaml

```diff
@@ -33,6 +34,13 @@ jobs:
             thanosruler: ""
     steps:
     - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+      with:
+        go-version: ${{ env.golang-version }}
+    - name: Build images
```

Comment:
Building images first will improve time-to-failure in case the code is incorrect.